core/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # 0.32.0 [unreleased]
 
+- Remove `Network`. `libp2p-core` is from now on an auxiliary crate only. Users
```

Comment:
This is an interesting take (the "auxiliary" part).

My PoV is that `libp2p-core` is / should remain a standalone foundation that should make it possible to write wire-compatible libp2p applications. i.e. the entire connection upgrade process etc that is currently in there is pretty foundational. Perhaps we agree on this and "auxiliary" was just not the best word to use here but if we don't agree then I'd like to discuss it :D

Reply:
Yes. I think we are on the same page. "Foundational" would have been a better word here.